.github/workflows/cd.yml

```diff
@@ -61,6 +61,7 @@ jobs:
 
       - name: Install wine64
         run: |
+          sudo apt update
```

Comment:
`sudo apt update -y`?

Reply:
no need
https://github.com/OfficeDev/TeamsFx/runs/5538024005?check_suite_focus=true 
this pipeline run success.